sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/AADOAuth2ClientConfiguration.java

```diff
@@ -31,7 +31,7 @@
 
     @Bean
     @ConditionalOnMissingBean
-    public AADClientRegistrationRepository clientRegistrationRepository(AADAuthenticationProperties properties) {
+    public ClientRegistrationRepository clientRegistrationRepository(AADAuthenticationProperties properties) {
```

Comment:
Not very necessary. a bean of `ClientRegistrationRepository ` is required when the oauth2 client dependency exists. it can be consistent with other bean definition styles. Should we keep this change?